unpick-format-utils/src/main/java/daomephsta/unpick/constantmappers/datadriven/parser/v2/UnpickV2Remapper.java

```diff
@@ -70,12 +90,50 @@ public void visitLineNumber(int lineNumber)
 
 	public void visitSimpleConstantDefinition(String group, String owner, String name, String value, String descriptor)
 	{
-		delegate.visitSimpleConstantDefinition(group, owner, name, value, descriptor);
+		//Reassigning the parameters tends to cause bugs
+		String remappedOwner = owner,
+			   remappedName = fieldMappings.getOrDefault(new FieldKey(owner, name), name),
+			   remappedDescriptor = descriptor;
+
+		if (classMappings.containsKey(owner))
+			remappedOwner = classMappings.get(owner);
```

Comment:
Can't you just do a `classMappings.getOrDefault(owner, owner)` when you declare `remappedOwner` local var?

Reply:
This was actually copied straight from the target remapping method, but yeah I switched everything to `getOrDefault`.